api/feature.go

```diff
@@ -76,7 +76,9 @@ func (f Feature) getVariantFromWeights(ctx *context.Context) *Variant {
 		return DISABLED_VARIANT
 	}
 
-	target := getNormalizedNumber(getSeed(ctx), f.Name, totalWeight)
+	stickiness := f.Variants[0].Stickiness
```

Comment:
This matches the [node client implementation](https://github.com/Unleash/unleash-client-node/blob/dda0aa9d75e2d979002d87b8f5b8f08e5dd91532/src/variant.ts#L88). The array is guaranteed to have a length of at least `1`, otherwise `totalWeight` would be `0` and the function would have already returned.